ios/RNTPTDocumentView.m

```diff
@@ -2590,8 +2596,10 @@ - (void)applyDocumentControllerSettings:(PTDocumentController *)documentControll
             UIBarButtonItem *bottomToolbarItem = [self itemForButton:bottomToolbarString
                                                     inViewController:documentController];
             if (bottomToolbarItem) {
-                [self ensureUniqueBottomBarButtonItem:bottomToolbarItem
-                                     inViewController:documentController];
+                [self removeLeftBarButtonItem:bottomToolbarItem
+                                inViewController:documentController];
+                [self removeRightBarButtonItem:bottomToolbarItem
+                                inViewController:documentController];
```

Comment:
You'll also want to `removeToolbarButtonItem:` here as well, to prevent the same item from appearing twice in the `toolbarItems`.

Reply:
added those lines